algorithms/soluciones/valid_parentheses.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+"""Valid_Parentheses.ipynb
+
+Automatically generated by Colab.
+
+Original file is located at
+    https://colab.research.google.com/drive/15tqvKTOrmKdX3n536yLORVmC_gNL8vSG
+"""
+
+def valid(s: str):
+    if len(s) >= 1 and len(s) <= 104:
+        per = [")", "(", "]", "[", "}", "{"]
+        if all(car in per for car in s):
+            tb = {")": "(", "]": "[", "}": "{"}
+            ver = []
+            for le in s:
+                if le in tb.values():
+                    ver.append(le)
+                elif le in tb:
+                    if not len(ver) > 0 or ver[-1] != tb[le]:
+                        return False, ",Error in the signs"
+                    else:
+                        ver.pop()
+                else:
+                    return False, ",Error Unexpected"
```

Comment:
Is it necessary? above in lines 11 and 13 you check the constraints, so what is the case where the code could be entered in this section.